views/index.erb

```diff
@@ -82,67 +82,67 @@
       <!--<div class="headline"><h2></h2></div>-->
 		  <ul class="slides">
 			  <li>
-          <a href="testimonials">
+          <a href="testimonials" aria-label="logotype">
             <img src="/images/testimonials/line.png" alt="" />
           </a>
```

Comment:
I'm not sure but setting `alt` to all images doesn't solve this issue?

Reply:
Unfortunately no.

Reply:
@ashie i can add not "logotype" => company